app/assets/javascripts/utilities/localDateTime.js

```diff
@@ -77,10 +77,21 @@ function timestampToLocalDateTimeLong(timestamp) {
 function timestampToLocalDateTimeShort(timestamp) {
   // example: "10 Dec 2018" if it is not the current year
   // example: "6 Sep" if it is the current year
+  // example: "22:43" if the date is the current date
 
   if (timestamp) {
-    const currentYear = new Date().getFullYear();
-    const givenYear = new Date(timestamp).getFullYear();
+    const dateToday = new Date();
```

Comment:
There is one change in the util function used here. It displays the time if the given date is the current date else it functions the same way

Reply:
I was not sure if I define another formatting function here that extends the current one or just edit the current one to include that change. I edited the current function for now to add that behaviour. any thoughts on this ?

Reply:
This behaviour will propagate in other places this function is used (Right now listing dashboard for creation date)

Reply:
cc @ludwiczakpawel as it's probably a product/design decision as well

Reply:
Ideally it would be nice to use relative time like:

> Just now, 2 mins ago, 1h ago, 5h ago, Yesterday, 2 days ago, Last week, ...

...but I imagine it's gonna be much more work, right? In that case, I think I'd default to use "Today" label IF given date is current date. BUT, as an addition we should add at least `title` attribute with full date and time like.

> March 17th, 4:30pm

Thoughts?

Reply:
I think it displays full date with time in title attribute now in the format "Friday, 11 September 2020, 07:59:08"

Reply:
But should it omit the time part and display the label "Today" if the date is current date?

Reply:
GitHub and Basecamp use this library (see our comments in here) <https://github.com/github/time-elements>.

It has localized components like `<time-ago>`, `<time-until>`, and the generic `<local-time>`.

I wonder if we should just include it and start using it everywhere @ludwiczakpawel @nickytonline - this is the size https://bundlephobia.com/result?p=@github/time-elements@3.1.1

Reply:
But I do agree that relative date labels looks nice (but it might be useful or cause confusion depending on the context it is rendered in).

Reply:
There is a util function there to already display time in relative labels in **app/assets/javascripts/utilities/timeAgo.js**

Reply:
> I wonder if we should just include it and start using it everywhere

Hell yes!

Reply:
@Rafi993 their time ago algorithm seems a bit more sofisticated than ours https://github.com/github/time-elements/blob/main/src/relative-time.js

Reply:
@rhymes you are right it does handle lot of cases. Also since it is a web component we can use it easily in preact and other places where there is js without preact (without a global util function which does need some extra things to be done when testing). It could possibly used in erb files too.

Reply:
I mean using in erb files without any additional setup.

Reply:
can I 
1. Alter this Pr to use https://github.com/github/time-elements in that one place and make sure it is ok 
2. Replace `dateTime` component with `time-element` in the next PR
3. Replace other places in subsequent PR

is that ok?

Reply:
It will add to our bundle size, but I imagine we'll also be removing code if use `time-elements` custom elements everywhere. As well, we won't need to worry about polyfills as the browsers we support support them. Let's try it out and we can go from there. Thanks for your patience @Rafi993. 😅